requirements.txt

```diff
@@ -11,6 +11,7 @@ flask-cors==3.0.9
 idna==2.9
 importlib-metadata==1.5.0
 importlib-resources==3.0.0
+jinja2
```

Comment:
Same here, do you think it is better not to have version and let python find the best.

I have mix feeling here.
On a different project, the pip version resolver was unable to resolve the version and did not find compatible versions for all the requirements and since we did not have the version documented we spent some time to retrieve them and I think eventually we were unable to make pip install -r requirements.txt happy althoungh an installation of the requirements package per package worked...

Reply:
Didn't actually mean for this commit to be included with this PR, I will add an explicit version with next PR (one that will actually use Jinja)

Reply:
Cool, thanks